apis/beacon/pool/sync_committees.yaml

```diff
@@ -0,0 +1,34 @@
+post:
+  operationId: submitSyncCommitteeSignatures
```

Comment:
Should it be `submitPoolSyncCommitteeSignatures` for consistency with other pool `operationId` items?

Reply:
updated